okhttp/src/main/java/io/grpc/okhttp/OkHttpTlsUpgrader.java

```diff
@@ -33,6 +33,7 @@
 
 import com.google.common.base.Preconditions;
 
+import com.squareup.okhttp.internal.tls.OkHostnameVerifier;
```

Comment:
There are _much_ worse "internal" classes we are using. The reason to copy it is so that we can upgrade it on our schedule instead of restricting the versions of okhttp our users can use.